build/generate_image_tags.sh

```diff
@@ -35,7 +35,7 @@ if [ $LATEST_GIT_TAG ]; then
 fi
 
 if [ $LATEST_TAG_REV ] && [ $LATEST_TAG_REV == $HEAD_REV ]; then
```

Comment:
Do these LATEST_TAG_REV also need to change to the git tag?

Reply:
no that should be correct since it compares revision(or sha)

Reply:
Ah I see you compare with the hashes but need to mark with the tag. Thx

/lgtm